src/models/cart.ts

```diff
@@ -8,7 +8,7 @@ export const CartsModel = sequelize.define('cart', {
     autoIncrement: true,
     primaryKey: true,
   },
-  product_name: {
+  product_name: { // should be id not name cuz its a FK 
```

Comment:
if it is something you intend to do you can add `TODO:` to clarify your intention, otherwise you communicate this error to your team in slack

Reply:
"I did that on Slack, and I also left a comment so they would notice it."